types/tx2/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for tx2 1.0
+// Project: https://github.com/pm2/tx2
+// Definitions by: Sasial <https://github.com/sasial-dev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import EventEmitter = require('node:events');
+
+interface ErrorObject {
+    name: string;
+    message: string;
+}
+
+interface Metric<T> {
+    val: () => T;
+    set: (data: T) => void;
+}
+
+interface Counter {
+    val: () => number;
+    inc: (amount: number) => void;
+    dec: (amount: number) => void;
+    reset: () => void;
+}
+
+declare class TX2 extends EventEmitter {
+    send(args: object): void;
```

Comment:
send is private:
https://github.com/pm2/tx2/blob/e0744395d4c9d10e55018eb36757ba5a6a1b8307/src/index.js#L40